docs/source/json_parser.py

```diff
@@ -0,0 +1,42 @@
+import json
+from pathlib import Path
+
+SOURCE_DIR = Path(__file__).parent
+info_json_file_path = SOURCE_DIR / "_tmp/" / "info.json"
+
+# To get all data needed for the release notes of one release,
+# we need to call `get_installer_information` for every _info.json
+# inside the release directory
+
+def get_installer_info(info_json_file_path: Path) -> dict:
+    """
+    Process _info.json file output by constructor to get installer information,
+    including the list of delivered packages
+    """
+    info_json_raw = json.loads(info_json_file_path.read_text())
+    info_dict = {key: info_json_raw[key] for key in info_json_raw.keys()}
+    
+    info_dict["pkgs_list_infos"] = [dist_str_to_dict(x) for x in info_json_raw["_dists"]]
+
+    return info_dict
+
+
+def dist_str_to_dict(dist: str) -> dict:
+        #  The `dist` strings are of the format "<name>-<version>-<hash>_<build_num>.conda"
+
+        pkg_name, pkg_version, pkg_split = dist.rsplit("-", 2)
+        pkg_hash = pkg_split.split("_")[0]
+        pkg_build_num = pkg_split.split("_")[1][0]
```

Comment:
```suggestion
        pkg_hash, pkg_build_num = pkg_split.split("_")
        # pop off ".conda" file extension
        pkg_build_num = int(pkg_build_num[:-6])
```

Your original code performs the splitting twice (less efficient) and also wouldn't work if we have a build number greater than 9. It's more robust to just pop off the `.conda` part.

Alternatively, you can also do the initial split with conda popped off:

`pkg_name, pkg_version, pkg_split = dist[:-6].rsplit("-", 2)`